saleor/settings.py

```diff
@@ -400,7 +400,9 @@ def get_host():
         ('product_small', 'thumbnail__60x60'),
         ('product_small_2x', 'thumbnail__120x120'),
         ('product_list', 'thumbnail__255x255'),
-        ('product_list_2x', 'thumbnail__510x510')]}
+        ('product_list_2x', 'thumbnail__510x510')],
+    'background_images': [
+        ('background_image', 'thumbnail__960x320')]}
```

Comment:
I think it should be more descriptive of where the image is used, how about `header_image` or something more descriptive of the purpose?